internal/cmd/cmds.go

```diff
@@ -142,6 +142,21 @@ func errInvalidValue(command, param string) error {
 	return fmt.Errorf("invalid value for a parameter in '%s' command for %s parameter", strings.ToUpper(command), strings.ToUpper(param))
 }
 
+//nolint:unparam
+func errIntegerOutOfRange(command string) error {
```

Comment:
given the error, we need not pass `command` to this. this should be a genetic `overflow` error.